src/Makefile.test_util.include

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2013-2019 The Bitcoin Core developers
+# Copyright (c) 2013-2023 The Bitcoin Core developers
```

Comment:
There's no need to manually bump dates, and they'll likely just be removed (#26817), so you can drop any changes like this.

Reply:
Done.